src/IECoreImage/ImageReader.cpp

```diff
@@ -519,6 +524,7 @@ CompoundObjectPtr ImageReader::readHeader()
 	m_implementation->updateHeader( header.get() );
 
 	header->members()["channelNames"] = new StringVectorData( cn );
+	header->members()["deep"] = new BoolData( m_implementation->isDeep() );
```

Comment:
Is doing this ok? I'm not clear on the distinction between the `header` and `blindData` for images.
For the way we are currently adding the `deep`, directly as a header data (just like the `channelNames`) I assume this is the correct fix for the test.

Reply:
Yeah, I think this makes sense.